### DISCUSSION TITLE
psutil

### DISCUSSION THREAD

**Discussion by xuying-92** [flet-dev/flet#3285] [Packages] [answered]
https://pypi.org/project/psutil/

**Comment by baseplate-admin** [comment]
I don’t get the meaning behind this request. `psutil` is not supported on android

**Comment by ndonkoHenri** [comment, marked as answer]
psutil 7.2.2 is now available on Android only: https://pypi.flet.dev/psutil
iOS cannot be supported for reasons mentioned here: https://github.com/flet-dev/mobile-forge/pull/71
